### PR TITLE
FW: Require turn assistant for nav modes using NAV_CTL_ALT

### PR DESCRIPTION
**Pull request by shellixyz**

### CODE REVIEW

File: src/main/navigation/navigation.c [hunk 1]
```diff
@@ -2775,7 +2775,7 @@ bool navigationRequiresTurnAssistance(void)
     const navigationFSMStateFlags_t currentState = navGetStateFlags(posControl.navState);
     if (STATE(FIXED_WING)) {
         // For airplanes turn assistant is always required when controlling position
-        return (currentState & NAV_CTL_POS);
+        return (currentState & (NAV_CTL_POS | NAV_CTL_ALT));
```
Comment (digitalentity) [inline src/main/navigation/navigation.c:2778]: Why is this necessary?

Comment (shellixyz) [reply src/main/navigation/navigation.c:2778]: I think it would be better for planes/wings. When trying to maintain altitude it is required to coordinate roll/pitch/yaw. It is not strictly necessary but it would probably give better experience "out of the box" meaning without the user enabling it manually.
